crates/top/re_sdk/src/lib.rs

```diff
@@ -46,13 +46,6 @@ pub fn default_server_addr() -> std::net::SocketAddr {
     std::net::SocketAddr::from(([127, 0, 0, 1], DEFAULT_SERVER_PORT))
 }
 
-/// The default amount of time to wait for the gRPC connection to resume during a flush
-#[allow(clippy::unnecessary_wraps)]
-pub fn default_flush_timeout() -> Option<std::time::Duration> {
-    // NOTE: This is part of the SDK and meant to be used where we accept `Option<std::time::Duration>` values.
-    Some(std::time::Duration::from_secs(3))
```

Comment:
The only default now is ∞, i.e. block until either success or failure. Anything else is arbitrary, and has to be opted-in too (which is easy enough to do).